templates/taxi/car_detail.html

```diff
@@ -1,12 +1,16 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>Car Details</h1>
+  <p>Id {{ car.id }}</p>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h2>Drivers</h2>
   <hr>
   <ul>
   {% for driver in car.drivers.all %}
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <a href="{% url 'taxi:car-update' pk=car.id %}">UPDATE CAR</a>
```

Comment:
The link to update the car is correctly implemented. Ensure that the update functionality is working as expected.